examples/README.md

```diff
@@ -2,20 +2,22 @@
 Riak Spark connector comes with several sample programs and demos:
 
 - [Simple Scala example](#simple-scala-example)
-- [Simple Scala RiakTS example](#simple-scala-riakts-example)
+- [Simple Scala Riak TS example](#simple-scala-riak-ts-example)
 - [Simple Scala DataFrame example](#simple-scala-dataframe-example)
-- [Simple Scala RiakTS DataFrame example](#simple-scala-riakts-dataframe-example)
+- [Simple Scala Riak TS DataFrame example](#simple-scala-riak-ts-dataframe-example)
 - [Simple Java example](#simple-java-example)
-- [Simple Java RiakTS example](#simple-java-riakts-example)
+- [Simple Java Riak TS example](#simple-java-riak-ts-example)
 - [OFAC demo](#ofac-demo)
-- [Scala RiakTS Parquet Example](#scala-riakts-parquet-example)
-- [Streaming Scala RiakKV Example](#streaming-scala-riakkv-example)
-- [Streaming Scala RiakTS Example](#streaming-scala-riakts-example)
+- [Scala RiakTS Parquet Example](#scala-riak-ts-parquet-example)
+- [Spark Streaming Examples](#spark-streaming-examples)
+  - [Spark Streaming Scala Riak KV Example](#spark-streaming-riak-kv-example)
+  - [Spark Streaming Scala Riak TS Example](#spark-streaming-riak-ts-example)
 
 It also comes with a helper for running a ready-to-go [bootstrapped Spark shell](#interactive-scala-shell).
```

Comment:
@korry8911 This line can be deleted since we're not mentioning the helper shell (src/main/repl/bin/rspark-shell) anymore. The examples just use run-example.